data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2707,6 +2707,21 @@
     [/filter_second]
 #enddef
 
+    # Filter how many times Malifor got attacked
+    [event]
+        name=attack end
+        first_time_only=no
+
```

Comment:
I think there should be a filter here for the player's units, too, so that the variable only gets incremented by ineffective attacks

Reply:
I agree, I missed the fact that Father Morvin could be attacking Malifor and triggering the line where he says he or Thera should be the ones to attack Malifor. Will fix.

Reply:
OK, looking forward to it.